packages/tpetra/core/src/Tpetra_Import_def.hpp

```diff
@@ -1281,8 +1281,7 @@ namespace Tpetra {
     // elements which it needs to send.
 #ifdef HAVE_TPETRA_MMM_TIMINGS
     prefix2 = std::string("Tpetra ")+ label + std::string(":iport_ctor:setupExport:4 ");
-    MM.release();
-    MM = rcp(new TimeMonitor(*TimeMonitor::getNewTimer(prefix2)));
+    MM = Teuchos::null; MM = rcp(new TimeMonitor(*TimeMonitor::getNewTimer(prefix2)));
```

Comment:
Yeah, `release` doesn't actually invoke the destructor, it just stops reference counting.  Thanks for the fix!